pandas/tests/test_nanops.py

```diff
@@ -1002,6 +1003,19 @@ def prng(self):
         return np.random.RandomState(1234)
 
 
+def test_add_overflow():
+    # see gh-14068
+    msg = "too (big|large) to convert"
+    with tm.assertRaisesRegexp(OverflowError, msg):
```

Comment:
@jreback : Ah, I think I see what you're saying from <a href="https://github.com/pydata/pandas/pull/14237#issuecomment-248757163">here</a>.  However, you're commenting on the wrong test.